### PR TITLE
Dependencies and codestyle update

### PR DESCRIPTION
**Pull request by MiroCillik**
Before fixing the "out of memory" problem I wanted to update this component to php 7 and recent coding style.

### CODE REVIEW

File: bootstrap.php [hunk 1]
```diff
@@ -1,18 +1,10 @@
...
-define('APP_NAME', 'ex-google-drive');
-if (!defined('ROOT_PATH')) {
-    define('ROOT_PATH', __DIR__);
-}
+declare(strict_types=1);
 
 date_default_timezone_set('Europe/Prague');
 
-ini_set('display_errors', true);
+ini_set('display_errors', '1');
```
Comment (odinuv) [inline bootstrap.php]: nic z toho co tu mas v bootstrapu by tam nemelo byt potreba

File: Dockerfile [hunk 2]
```diff
@@ -1,10 +1,21 @@
-FROM keboola/base-php56
-MAINTAINER Miro Cillik <miro@keboola.com>
+FROM php:7-cli
```
Comment (odinuv) [inline Dockerfile]: dal bych tam 7.4 nebo 7.3 - v composer.json mas 7.3

**Review by odinuv — APPROVED** [review]
no pekna vykopavka, nezkousel jsem to teda spoustet, tak snad to jede :)

**Comment by odinuv** [comment]
jeste takova drobnost:

> phpunit --coverage-clover build/logs/clover.xml --coverage-xml=build/logs/coverage-xml --log-junit=build/logs/phpunit.junit.xml
PHPUnit 8.5.2 by Sebastian Bergmann and contributors.
Error:         No code coverage driver is available

**Comment by odinuv** [comment]
a nemel by to reviewovat spis components team?

**Comment by MiroCillik** [comment]
Diky za komenty. Asi by to mal reviewovat niekto z components teamu, ale toto som chcel mat posvatene od niekoho zkusenejsieho.

File: bootstrap.php (hunk 1 above)
Comment (MiroCillik) [reply bootstrap.php]: ani ten error handler?

Comment (odinuv) [reply bootstrap.php]: imho to tam bylo kvuli nejakymu prastarymu phpunitu

**Pull request merged**